recipes-support/lshw/files/ldflags.patch

```diff
@@ -1,6 +1,6 @@
-diff -Naur /home/anton/lshw-old/src/Makefile lshw-B.02.16/src/Makefile
+diff -Naur /home/anton/lshw-old/src/Makefile lshw-B.02.18/src/Makefile
 --- /home/anton/lshw-old/src/Makefile	2017-02-07 16:21:52.554738182 +0100
-+++ lshw-B.02.16/src/Makefile	2017-02-07 16:22:45.578588072 +0100
++++ lshw-B.02.18/src/Makefile	2017-02-07 16:22:45.578588072 +0100
```

Comment:
Should these be `lshw-B.02.17`, not `.18`?

Reply:
Yes, I tried with the .18 first, but Yocto installed the .17 (????) so I just replaced my modifications with .17. Looks like I forgot this one.